src/components/Game/Letters/Letters.scss

```diff
@@ -0,0 +1,30 @@
+@import "../../../styles/abstracts/variables";
+
+.Letters {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.LettersRow {
+    display: flex;
+    justify-content: center;
+}
+
+.Letter {
+    height: 55px;
+    width: 50px;
+    margin: 10px;
+    border: 1px solid $color-black;
+    border-radius: 5px;
+    text-transform: uppercase;
+    text-align: center;
+    font-size: 40px;
+    line-height: 50px;
```

Comment:
55px, 50px, 10px, 40px, 50px - are there any semantic relation between all these values
if this is the case consider introducing a variable scoped to this style ruleset, e.g. $size

Reply:
🤔 semantic relationship doesn't convince me enough. i'd sooner see in it a set of sizes these values make for a Letter. i introduced some variables and mixins.